week_1/2_apr/youtube-fe/styles/header.css

```diff
@@ -0,0 +1,100 @@
+.header {
+    height: 55px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.left-section {
+    display: flex;
+    align-items: center;
+}
+
+.hamburger-menu {
+    height: 24px;
+    margin-left: 24px;
```

Comment:
@codal-kcolaco also check how we can give margin left right in one line or if any other margin as with padding

Reply:
@abhi-codal Yes, will do